buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -4,9 +4,10 @@ object Versions {
     const val COROUTINES = "1.7.3"
     const val ANDROID_GRADLE_PLUGIN = "7.4.2"
     const val JUNIT = "4.13.2"
-    const val ROBOLECTRIC = "4.3"
-    const val MOCKITO = "4.11.0"
-    const val MOCKITO_KOTLIN = "4.1.0"
+    const val ROBOLECTRIC = "4.10.3"
```

Comment:
Hmm. Is that Robolectric upgrade came from the `master` branch?

Reply:
Yes and no. Mockito branch was started from Robolectric one. I've synced it with `master`.